### DISCUSSION TITLE
Deploy as a part of already existing website

### DISCUSSION THREAD

**Discussion by tadjik1** [stackblitz/tutorialkit#243] [Q&A] [answered]
Hey there,
thanks a lot for such a fantastic tool, I've been able to create quite a comprehensive tutorial with lots of practical exercises in "almost" no time!
But now, I'm stuck a bit with the deployment. The problem here is that I already have website, and I want tutorial be part of it. Is that even possible? 
I like everything about the current layout and don't want to change a thing, would it be possible to include it as an iframe? Or modify the template, so in the "build" I would only have content part (w/o html and all the rest element of the page)

**Comment by Nemikolh** [comment, marked as answer]
It is! 😃

You can deploy it under a sub path so that you serve your tutorial at `https://example.com/subpath`.

To do so you need to modify your astro config like this:

```ts
export default defineConfig({
  base: '/subpath' 
  //... 
}) 
```

To learn more you can check https://docs.astro.build/en/reference/configuration-reference/#base

You will also have to make sure to serve the assets under that sub path with the correct COOP COEP headers. See https://tutorialkit.dev/guides/deployment/#headers-configuration to learn more.

If you have any other questions, let us know!

**Reply by tadjik1** [reply]
thank you so much!
